gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeConfig.ts

```diff
@@ -1,57 +1,61 @@
 import { Result } from '@chainlink/gauntlet-core'
 import { logger, BN, prompt } from '@chainlink/gauntlet-core/dist/utils'
-import { SolanaCommand, TransactionResponse, RawTransaction } from '@chainlink/gauntlet-solana'
-import { AccountMeta, PublicKey } from '@solana/web3.js'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
 import { ORACLES_MAX_LENGTH } from '../../../lib/constants'
 import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
-import { getRDD } from '../../../lib/rdd'
-import { makeTx } from '../../../lib/utils'
+import RDD from '../../../lib/rdd'
 
 type Input = {
   oracles: {
     signer: string
     transmitter: string
   }[]
   f: number | string
+  proposalId: string
 }
-export default class SetConfig extends SolanaCommand {
-  static id = 'ocr2:set_config'
+
+export default class ProposeConfig extends SolanaCommand {
+  static id = 'ocr2:propose_config'
   static category = CONTRACT_LIST.OCR_2
 
   static examples = [
-    'yarn gauntlet ocr2:set_config --network=devnet --state=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC',
+    'yarn gauntlet ocr2:propose_config --network=devnet --rdd=[PATH_TO_RDD] --proposalId=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC [AGGREGATOR_ADDRESS]',
```

Comment:
This doesn't quite make sense to me, you're operating on the proposal (the "object" of the command) so I'd pass proposal as the argument and use a flag for the state. Some of these `propose` commands don't even need a `state`/aggregator address passed in

Reply:
For example in the future we could offer specifying oracles from a file, in which case there would be no `state` parameter to pass here

Reply:
Would be good if we had nested commands:

```
gauntlet ocr2 aggregator create <id> --flags
gauntlet ocr2 proposal update <proposal_id> --flags
```

etc

Reply:
I see your point, but from a user perspective, where gauntlet is operating I don't think is relevant, but more to have a consistent user experience. If every `ocr2` command accepts an OCR account address as argument, I think it should be like that in every case. 

> Would be good if we had nested commands:
> 
> ```
> gauntlet ocr2 aggregator create <id> --flags
> gauntlet ocr2 proposal update <proposal_id> --flags
> gauntlet ocr2 migrate
> ```
> 
> etc

If in the future we implement this idea, we'll can do it there